support-services/src/main/scala/com/gu/zuora/ZuoraServiceProvider.scala

```diff
@@ -0,0 +1,15 @@
+package com.gu.zuora
+
+import com.gu.okhttp.RequestRunners.configurableFutureRunner
+import com.gu.support.config.{ZuoraConfig, ZuoraConfigProvider}
+import com.gu.support.touchpoint.TouchpointServiceProvider
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.DurationInt
+
+class ZuoraServiceProvider(configProvider: ZuoraConfigProvider)(implicit ec: ExecutionContext)
+  extends TouchpointServiceProvider[ZuoraService, ZuoraConfig](configProvider){
```

Comment:
we use a ZuoraSubscribeService for the few operations that are needed in the backend, to make mocking easier/safer.  In this case, we would presumably only need one operation, so it makes sense to have a ZuoraLookupService  in a similar way if we want to share the actual ZuoraService class.

Reply:
This is a good point. I've gone back and broken out the functionality related to gifting into a separate class which I'd been wanting to do anyway as `ZuoraService` was getting much too big. 
I've also created a trait `ZuoraGiftLookupService` which just has the lookup method on it and this is all that now gets exposed in support-frontend

Reply:
great work breaking up zuoraservice, yes we have more classes now, but each one is more cohesive  👏